aviary/subsystems/propulsion/motor/model/motor_mission.py

```diff
@@ -110,14 +113,20 @@ def setup(self):
                 has_diag_partials=True,
             ),
             promotes_inputs=[
-                ('max_torque', Aircraft.Engine.Motor.TORQUE_MAX),
+                ('max_torque', Dynamic.Mission.TORQUE_MAX),
```

Comment:
is max torque actually dynamic? this seems like it would be constant for a particular motor

Reply:
or is this variable intended to be generic for motors and engines?

Reply:
Here max torque is referring to the same condition as THRUST_NET_MAX, SHAFT_POWER_MAX, etc. where it is output using max throttle